lib/handler/mruby.c

```diff
@@ -159,6 +159,11 @@ mrb_value h2o_mruby_compile_code(mrb_state *mrb, h2o_mruby_config_vars_t *config
         abort();
     }
 
+    /* reset configuration context */
+    h2o_mruby_eval_expr(mrb, "H2O::ConfigurationContext.instance = H2O::ConfigurationContext.new");
```

Comment:
Maybe it would be better to introduce a `reset` method to `H2O::ConfigurationContext` and call it, instead of having duplicate logic at here and t/00unit.mmruby/acl.rb.